index.js

```diff
@@ -8,27 +8,15 @@ module.exports = function randomAccessKeychain(service, account) {
   // We store the password in base64, because we can not store null bytes otherwise.
 
   function getPassword(cb) {
-    keychain.getPassword(
-      {
-        account: account,
-        service: service,
-      },
-      function(err, password) {
-        if (err) return cb(err)
-        cb(null, Buffer.from(password, 'base64'))
-      }
-    )
+    keytar.getPassword(service, account)
+      .then(password => cb(null, Buffer.from(password, 'base64')))
```

Comment:
`Buffer.from` should complain if this `keytar.getPassword` returns null, so we will always call the error callback - I believe that this isn't what you intended, given the changes you made to the `getPassword` function below.